diff --git a/hover/util/log.go b/hover/util/log.go
index ea025ea..6abbe9f 100644
--- a/hover/util/log.go
+++ b/hover/util/log.go
@@ -15,7 +15,6 @@
 package util

 import (
-       "io/ioutil"
        "log"
        "os"
 )
@@ -28,7 +27,7 @@ var (
 )

 func logInit() {
-       Debug = log.New(ioutil.Discard, "DBG: ", log.Ldate|log.Ltime|log.Lshortfile)
+       Debug = log.New(os.Stdout, "DBG: ", log.Ldate|log.Ltime|log.Lshortfile)
        Info = log.New(os.Stdout, "INF: ", log.Ldate|log.Ltime)
        Warn = log.New(os.Stdout, "WRN: ", log.Ldate|log.Ltime)
        Error = log.New(os.Stderr, "ERR: ", log.Ldate|log.Ltime)
